### PR TITLE
Fixes, Tweaks and Balance

### LINKED ISSUE #432: BUG: Closests can fit in disposal units
#### Description of issue
- See the title.

#### Difference between expected and actual behavior
- It should not be possible to pull a large locker into a disposal unit.

### LINKED ISSUE #601: BUG : Found a Baycode testing area.
#### Description of issue:

During an escape, myself and the SSO were told that too many people were on the escape pod after it fired and that it would exlode if people didnt jump, so we did the hero thing and jumped, drifting for a while to the north, till we hit a wall. I used the wall itself and a fire extinguisher to follow the wall to the north, where I found a giant flat area with O2 and gravity, which was likely, as guessed by TrialMod, TheLion(?), a Baycode testing area. I was asked to make a report on this by the TrialMod.

#### Difference between expected and actual behavior:
I dont know if thats supposed to be accessible, but according to the mod, its been disused for a good while.

#### Steps to reproduce:
Fire escape pod, jump out, drift north, follow wall to the north, profit, safe, O2, Gravity, correct pressure.

#### What do you think it'd add:
It was my first Major Human Victory. Feels good, man.

### LINKED ISSUE #607: Runes Aren't Cleanable
Various bloodrunes created by signal abilities can't be cleaned off

### LINKED ISSUE #655: [Suggestion] Nerf alcohol damage
#### Suggestion:
Currently, drinking an entire alcohol bottle will pretty much kill you and you cannot stop it unless medical is somewhat competent, literally an entire bottle will kill your liver leading this to kidney failure and then complete organ failure, killing you in less than 5 minutes.
I thought about nerfing how much alcohol damages the liver to some acceptable levels so it doesn't kill your liver with one bottle in such low time.

#### What do you think it'd add:
Less death by alcohol

### PR DESCRIPTION
**Pull request by JakeDaBoss**
Closes #607 
Closes #432 
Closes #601 
Closes #655

### CODE REVIEW

**Comment by Snypehunter007** [comment]
Mapping changes need to be reverted, I've got a branch of mapping fixes as well.

**Comment by JakeDaBoss** [comment]
Will do once I get home.

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -675,23 +675,19 @@ var/list/admin_verbs_mentor = list(
 	set desc = "Cause an explosion of varying strength at your location."
 
 	var/turf/epicenter = mob.loc
-	var/list/choices = list("Small Bomb", "Medium Bomb", "Big Bomb", "Huge Bomb", "Colossal Bomb", "Apocalyptic Bomb", "Custom Bomb")
```
Comment (NanakoAC) [inline code/modules/admin/admin_verbs.dm]: why would you do this, I added these myself, they're for largescale testing

Comment (NanakoAC) [reply code/modules/admin/admin_verbs.dm]: please revert these changes, this is an admin tool

File: code/modules/mining/mine_turfs.dm [hunk 2]
```diff
@@ -430,7 +431,7 @@ var/list/mining_floors = list()
 /turf/simulated/mineral/random
 	name = "Mineral deposit"
 	var/mineralSpawnChanceList = list("Uranium" = 5, "Platinum" = 5, "Iron" = 35, "Carbon" = 35, MATERIAL_DIAMOND = 1, MATERIAL_GOLD = 5, MATERIAL_SILVER = 5, MATERIAL_PHORON = 10)
-	var/mineralChance = 1.5 //means 1% chance of this plot changing to a mineral deposit
+	var/mineralChance = 25 //means 25% chance of this plot changing to a mineral deposit
```
Comment (NanakoAC) [inline code/modules/mining/mine_turfs.dm:434]: 25% is a completely crazy mineral chance, have you tested this? need to see it in action. Is there some major reworking to what the number means?

**Review by NanakoAC — CHANGES_REQUESTED** [review]
concenrns about ore distribution and the reversion of an admin verb

File: code/modules/admin/admin_verbs.dm (hunk 1 above)
Comment (JakeDaBoss) [reply code/modules/admin/admin_verbs.dm]: @Snypehunter007 asked me to remove them.

File: code/modules/mining/mine_turfs.dm (hunk 2 above)
Comment (JakeDaBoss) [reply code/modules/mining/mine_turfs.dm:434]: It's not too bad, honestly. 25% is also lore appropriate. You know Aegis 7 was a literal goldmine right?

File: code/modules/admin/admin_verbs.dm (hunk 1 above)
Comment (JakeDaBoss) [reply code/modules/admin/admin_verbs.dm]: And also, use custom bomb. Jfc.

Comment (JakeDaBoss) [reply code/modules/admin/admin_verbs.dm]: Third note: This was requested by Snype. I adhere to his requests.

Comment (NanakoAC) [reply code/modules/admin/admin_verbs.dm]: i'll talk to snype about it, put them back please

Comment (JakeDaBoss) [reply code/modules/admin/admin_verbs.dm]: 18:17] Snypehunter007: Thats fine then.

As agreed upon. Resolved.

**Review by NanakoAC — APPROVED** [review]

**Pull request merged**